### PR TITLE
Replacing arn:aws with arn:${AWS::Partition} in CF Templates

### PR DESCRIPTION
**Pull request by waynekata**
PR for https://github.com/stelligent/mu/issues/348

### CODE REVIEW

File: templates/assets/service-iam.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ Resources:
           - sts:AssumeRole
       Path: "/"
       ManagedPolicyArns:
-      - arn:aws:iam::aws:policy/service-role/AWSCodeDeployRole
+      - arn:${AWS::Partition}:iam::aws:policy/service-role/AWSCodeDeployRole
```
Comment (cplee) [inline templates/assets/service-iam.yml]: need `!Sub`

File: templates/assets/service-iam.yml [hunk 2]
```diff
@@ -155,7 +155,7 @@ Resources:
           - sts:AssumeRole
       Path: "/"
       ManagedPolicyArns:
-      - arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess
+      - arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ReadOnlyAccess
```
Comment (cplee) [inline templates/assets/service-iam.yml]: need `!Sub`

File: templates/assets/service-iam.yml [hunk 3]
```diff
@@ -300,7 +300,7 @@ Resources:
           - sts:AssumeRole
       Path: "/"
       ManagedPolicyArns:
-      - arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole
+      - arn:${AWS::Partition}:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole
```
Comment (cplee) [inline templates/assets/service-iam.yml]: need `!Sub`

**Review by cplee — CHANGES_REQUESTED** [review]
there are a few spots that are missing `!Sub`

**Comment by waynekata** [comment]
Working on fixing this

**Review by cplee — APPROVED** [review]

**Comment by cplee** [comment]
close #348

**Pull request merged**